components/compliance-service/api/profiles/server/migrate.go

```diff
@@ -67,8 +67,7 @@ func (srv *PGProfileServer) migrateDiskProfiles() error {
 
 			_, err = srv.storeProfile(namespace, tmpFile)
 			if err != nil {
-				logrus.WithError(err).Errorf("Could not store profile %q", tmpFile)
-				finalError = multierr.Combine(finalError, err)
```

Comment:
Removing this line is the important part of this PR. Prior to this PR, the process for migrating profiles was like this:
* Look for any profiles in the directory used to store profiles in A1.
* For any profile found, attempt to store the profile the way A2 does.
* If it fails, record an error and fail the entire migration once all profiles have had a migration attempt.
* If if succeeds, archive the profile (ie, move it somewhere else on disk).

After this PR, if a profile fails to migrate, an error message will be logged but entire migration process will succeed. This has the effect of leaving profiles that failed to migrate where they are, so that when compliance-service is restarted any remaining profiles will be migrated again. Alternatively, the process could change so that profiles are archived if their migration fails or succeeds, so that users won't see error messages after the first time.

Reply:
If it's not too difficult, I might prefer a third option of archiving the "bad" profiles to a "bad profile zone" so users can identify them as not-migrate-able and try to fix them. But if it's a non-trivial amount of work then it's probably not worth it, as I understand it.

Reply:
I think that is a good idea, but since we expose the existing profile store and archive directories as config parameters, etc it would be a non-trivial amount of work to get that going, and preventing the service from crashing on upgrade is a higher priority.